io.github.wxmaxima_developers.wxMaxima.yml

```diff
@@ -0,0 +1,75 @@
+app-id: io.github.wxmaxima_developers.wxMaxima
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: wxmaxima
+finish-args:
+      - --share=ipc
+      - --socket=x11
+      - --env=LC_CTYPE="en_US.UTF-8"
+      - --filesystem=home:rw
```

Comment:
Is full user home dir access really required?

Reply:
Access to the file system is necessary to load saved sessions or to load scripts, save numerical results or images. Although you can restrict access to a specific directory, this would detract from the user experience.